optimization/optimizer.py

```diff
@@ -19,13 +21,14 @@ def load_course_map(course_name="COTA"):
     :return elev_profile: map of the elevations on a course
     """
     if course_name == "COTA":
-        with open('COTAelevation.txt', 'r') as file:
+        with open('COTAelevation_var.txt', 'r') as file:
             line = file.readline()
             count = 1
             data = []
             while line and count < 3:
                 count += 1
                 line = file.readline()
+            ydir = float(line.split(" ")[1][0])
```

Comment:
In this implementation ydir does not change for the entire profile. Should it be different with each step, or is it supposed to be the same always.